### PR TITLE
Drop python 3.5 support

### PR DESCRIPTION
**Pull request by saroad2**
Finally, dropping support for Python 3.5, as discussed in beeware/toga#1199

## PR Checklist:

- [X] All new features have been tested
- [ ] All new features have been documented
- [X] I have read the **CONTRIBUTING.md** file
- [x] I will abide by the code of conduct

### CODE REVIEW

**Review by phildini — APPROVED** [review]
lgtm, not sure if we're sold on dropping 3.5, but if we are, let's go!

**Comment by saroad2** [comment]
Thanks, @phildini ! 

As we talked in beeware/toga#1199, we are currently working on dropping Python 3.5 support on all of Beeware's repositories :) Toga and Briefcase already dropped 3.5 support.

Just waiting for @freakboy3742 approval for this PR.

**Review by freakboy3742 — APPROVED** [review]
👍 The king is dead; long live the king...

**Pull request merged**
